open_city_profile/schema.py

```diff
@@ -0,0 +1,17 @@
+import graphene
+import graphql_jwt
+
+import profiles.schema
+
+
+class Query(profiles.schema.Query, graphene.ObjectType):
+    pass
+
+
+class Mutation(profiles.schema.Mutation, graphene.ObjectType):
+    token_auth = graphql_jwt.ObtainJSONWebToken.Field()
```

Comment:
we are not actually issuing any tokens, so all of this are redundant. just having authentication middleware should suffice for our purposes